config/job.js

```diff
@@ -4,6 +4,44 @@ const Annotations = require('./annotations');
 const Joi = require('joi');
 const Regex = require('./regex');
 
+// ref. https://github.com/hapijs/joi/blob/v13.3.0/API.md#extendextension
+const sdJoi = Joi.extend(joi => ({
+    base: joi.string(),
+    name: 'string',
+    language: {
+        commitBranch: 'needs to be a commit branch string/regex'
+    },
+    rules: [
+        {
+            name: 'commitBranch',
+            validate(params, value, state, options) {
+                const specificBranchPos = 3;
```

Comment:
I think it's more clear this is a constant, hardcoded value if you put it in all caps like https://github.com/screwdriver-cd/executor-k8s-vm/blob/master/index.js#L14

Reply:
I fixed!